src/main/kotlin/org/rust/lang/core/psi/ext/RsStructItem.kt

```diff
@@ -34,6 +35,10 @@ val RsStructItem.kind: RsStructKind get() {
 
 val RsStructItem.isTupleStruct get() = tupleFields != null
 
+val RsStructItem.fieldTypes: List<Ty>
+    get() =
+        this.blockFields?.fieldDeclList?.mapNotNull { it.typeReference?.type }.orEmpty()
```

Comment:
Struct also can have tuple fields. Let's generalize it with `RsEnumVariant.fieldTypes` around `RsFIeldsOwner`